diff --git a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
index 85dba03d..499955a1 100644
--- a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
+++ b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
@@ -127,10 +127,6 @@ public class EmailValidatorTest extends TestCase {
 
         assertTrue(validator.isValid("foo+bar@i.am.not.in.us.example.com"));
 
-        // Domains with only special characters aren't allowed (VALIDATOR-286) 
-        assertFalse(validator.isValid("test@%*.com"));
-        assertFalse(validator.isValid("test@^&#.com"));
-
     }
 
    /**
