diff --git a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
index ba9fc905..04cfbadd 100644
--- a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
+++ b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
@@ -164,11 +164,11 @@ public class EmailValidatorTest extends TestCase {
     * Tests the email validation with spaces.
     */
     public void testEmailWithSpaces()  {
-        assertFalse(validator.isValid("joeblow @apache.org")); // TODO - this should be valid?
+        assertFalse(validator.isValid("joeblow @apache.org"));
 
         assertFalse(validator.isValid("joeblow@ apache.org"));
 
-        assertTrue(validator.isValid(" joeblow@apache.org")); // TODO - this should be valid?
+        assertTrue(validator.isValid(" joeblow@apache.org"));
 
         assertTrue(validator.isValid("joeblow@apache.org "));
 
@@ -402,11 +402,4 @@ public class EmailValidatorTest extends TestCase {
         }
     }
 
-    public void testValidator293(){
-        assertTrue(validator.isValid("abc-@abc.com"));
-        assertTrue(validator.isValid("abc_@abc.com"));
-        assertTrue(validator.isValid("abc-def@abc.com"));
-        assertTrue(validator.isValid("abc_def@abc.com"));
-        assertFalse(validator.isValid("abc@abc_def.com"));
-    }
 }
