diff --git a/src/main/java/org/apache/commons/validator/routines/DomainValidator.java b/src/main/java/org/apache/commons/validator/routines/DomainValidator.java
index 2f0a9dfd..11cfd442 100644
--- a/src/main/java/org/apache/commons/validator/routines/DomainValidator.java
+++ b/src/main/java/org/apache/commons/validator/routines/DomainValidator.java
@@ -65,13 +65,8 @@ public class DomainValidator implements Serializable {
     private static final long serialVersionUID = -4407125112880174009L;
 
     // Regular expression strings for hostnames (derived from RFC2396 and RFC 1123)
-
-    // RFC2396: domainlabel   = alphanum | alphanum *( alphanum | "-" ) alphanum
     private static final String DOMAIN_LABEL_REGEX = "\\p{Alnum}(?>[\\p{Alnum}-]*\\p{Alnum})*";
-
-    // RFC2396 toplabel = alpha | alpha *( alphanum | "-" ) alphanum
-    private static final String TOP_LABEL_REGEX = "\\p{Alpha}|(?:\\p{Alpha}(?:[\\p{Alnum}-])*\\p{Alnum})";
-
+    private static final String TOP_LABEL_REGEX = "\\p{Alpha}{2,}";
     private static final String DOMAIN_NAME_REGEX =
             "^(?:" + DOMAIN_LABEL_REGEX + "\\.)+" + "(" + TOP_LABEL_REGEX + ")$";
 
@@ -142,12 +137,6 @@ public class DomainValidator implements Serializable {
         return allowLocal && hostnameRegex.isValid(domain);
     }
 
-    // package protected for unit test access 
-    final boolean isValidDomainSyntax(String domain) {
-        String[] groups = domainRegex.match(domain);
-        return (groups != null && groups.length > 0);
-    }
-
     /**
      * Returns true if the specified <code>String</code> matches any
      * IANA-defined top-level domain. Leading dots are ignored if present.
