diff --git a/src/main/java/org/apache/commons/validator/routines/DomainValidator.java b/src/main/java/org/apache/commons/validator/routines/DomainValidator.java
index 17198013..44f74064 100644
--- a/src/main/java/org/apache/commons/validator/routines/DomainValidator.java
+++ b/src/main/java/org/apache/commons/validator/routines/DomainValidator.java
@@ -67,12 +67,10 @@ public class DomainValidator implements Serializable {
     // Regular expression strings for hostnames (derived from RFC2396 and RFC 1123)
 
     // RFC2396: domainlabel   = alphanum | alphanum *( alphanum | "-" ) alphanum
-    // Max 63 characters
-    private static final String DOMAIN_LABEL_REGEX = "\\p{Alnum}(?>[\\p{Alnum}-]{0,61}\\p{Alnum})?";
+    private static final String DOMAIN_LABEL_REGEX = "\\p{Alnum}(?>[\\p{Alnum}-]*\\p{Alnum})*";
 
     // RFC2396 toplabel = alpha | alpha *( alphanum | "-" ) alphanum
-    // Max 63 characters
-    private static final String TOP_LABEL_REGEX = "\\p{Alpha}(?>[\\p{Alnum}-]{0,61}\\p{Alnum})?";
+    private static final String TOP_LABEL_REGEX = "\\p{Alpha}|(?:\\p{Alpha}(?:[\\p{Alnum}-])*\\p{Alnum})";
 
     private static final String DOMAIN_NAME_REGEX =
             "^(?:" + DOMAIN_LABEL_REGEX + "\\.)+" + "(" + TOP_LABEL_REGEX + ")$";
@@ -137,9 +135,6 @@ public class DomainValidator implements Serializable {
      * @return true if the parameter is a valid domain name
      */
     public boolean isValid(String domain) {
-        if (domain == null || domain.length() > 253) {
-            return false;
-        }
         String[] groups = domainRegex.match(domain);
         if (groups != null && groups.length > 0) {
             return isValidTld(groups[0]);
@@ -147,12 +142,8 @@ public class DomainValidator implements Serializable {
         return allowLocal && hostnameRegex.isValid(domain);
     }
 
-    // package protected for unit test access
-    // must agree with isValid() above
+    // package protected for unit test access 
     final boolean isValidDomainSyntax(String domain) {
-        if (domain == null || domain.length() > 253) {
-            return false;
-        }
         String[] groups = domainRegex.match(domain);
         return (groups != null && groups.length > 0);
     }
