diff --git a/src/test/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigitTest.java b/src/test/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigitTest.java
index 4ee88052..f91661ee 100644
--- a/src/test/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigitTest.java
+++ b/src/test/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigitTest.java
@@ -51,24 +51,4 @@ public class CUSIPCheckDigitTest extends AbstractCheckDigitTest {
         invalid = new String[] {"0378#3100"};
     }
 
-    private static String invalidCheckDigits[] = {"DUS0421CW",
-                                                  "DUS0421CN",
-                                                  "DUS0421CE"
-    };
-
-    public void testVALIDATOR_336_InvalidCheckDigits() {
-        for (int i = 0; i < invalidCheckDigits.length; i++) {
-            String invalidCheckDigit = invalidCheckDigits[i];
-            assertFalse("Should fail: " + invalidCheckDigit, routine.isValid(invalidCheckDigit));
-        }
-    }
-
-    private static String validCheckDigits[] = {"DUS0421C5"};
-
-    public void testVALIDATOR_336_ValidCheckDigits() {
-        for (int i = 0; i < validCheckDigits.length; i++) {
-            String validCheckDigit = validCheckDigits[i];
-            assertTrue("Should fail: " + validCheckDigit, routine.isValid(validCheckDigit));
-        }
-    }
 }
