diff --git a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
index 61721d35..391070cc 100644
--- a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
+++ b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
@@ -136,12 +136,7 @@ public class EmailValidatorTest extends TestCase {
 
     }
 
-    public void testVALIDATOR_315() {
-        assertFalse(validator.isValid("me@at&t.net"));
-        assertTrue(validator.isValid("me@att.net")); // Make sure TLD is not the cause of the failure
-    }
-
-    /**
+   /**
     * Tests the email validation with commas.
     */
     public void testEmailWithCommas()  {
