diff --git a/src/main/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigit.java b/src/main/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigit.java
index 56258d82..0e26f1ce 100644
--- a/src/main/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigit.java
+++ b/src/main/java/org/apache/commons/validator/routines/checkdigit/CUSIPCheckDigit.java
@@ -61,18 +61,16 @@ public final class CUSIPCheckDigit extends ModulusCheckDigit {
      *
      * @param character The character to convert
      * @param leftPos The position of the character in the code, counting from left to right
-     * @param rightPos The position of the character in the code, counting from right to left
+     * @param rightPos The positionof the character in the code, counting from right to left
      * @return The integer value of the character
      * @throws CheckDigitException if character is not alphanumeric
      */
     protected int toInt(char character, int leftPos, int rightPos)
             throws CheckDigitException {
         int charValue = Character.getNumericValue(character);
-        // the final character is only allowed to reach 9
-        final int charMax = rightPos == 1 ? 9 : 35;
-        if (charValue < 0 || charValue > charMax) {
+        if (charValue < 0 || charValue > 35) {
             throw new CheckDigitException("Invalid Character[" +
-                    leftPos + "," + rightPos + "] = '" + charValue + "' out of range 0 to " + charMax);
+                    leftPos + "] = '" + charValue + "'");
         }
         return charValue;
     }
