diff --git a/src/main/java/org/apache/commons/validator/routines/EmailValidator.java b/src/main/java/org/apache/commons/validator/routines/EmailValidator.java
index 26f689ec..ec864d53 100644
--- a/src/main/java/org/apache/commons/validator/routines/EmailValidator.java
+++ b/src/main/java/org/apache/commons/validator/routines/EmailValidator.java
@@ -51,32 +51,18 @@ public class EmailValidator implements Serializable {
     private static final Pattern USER_PATTERN = Pattern.compile(USER_REGEX);
 
     private final boolean allowLocal;
-    private final boolean allowTld;
 
     /**
      * Singleton instance of this class, which
      *  doesn't consider local addresses as valid.
      */
-    private static final EmailValidator EMAIL_VALIDATOR = new EmailValidator(false, false);
-
-    /**
-     * Singleton instance of this class, which
-     *  doesn't consider local addresses as valid.
-     */
-    private static final EmailValidator EMAIL_VALIDATOR_WITH_TLD = new EmailValidator(false, true);
+    private static final EmailValidator EMAIL_VALIDATOR = new EmailValidator(false);
 
     /**
      * Singleton instance of this class, which does
      *  consider local addresses valid.
      */
-    private static final EmailValidator EMAIL_VALIDATOR_WITH_LOCAL = new EmailValidator(true, false);
-
-
-    /**
-     * Singleton instance of this class, which does
-     *  consider local addresses valid.
-     */
-    private static final EmailValidator EMAIL_VALIDATOR_WITH_LOCAL_WITH_TLD = new EmailValidator(true, true);
+    private static final EmailValidator EMAIL_VALIDATOR_WITH_LOCAL = new EmailValidator(true);
 
     /**
      * Returns the Singleton instance of this validator.
@@ -94,42 +80,11 @@ public class EmailValidator implements Serializable {
      * @param allowLocal Should local addresses be considered valid?
      * @return singleton instance of this validator
      */
-    public static EmailValidator getInstance(boolean allowLocal, boolean allowTld) {
+    public static EmailValidator getInstance(boolean allowLocal) {
         if(allowLocal) {
-            if (allowTld) {
-                return EMAIL_VALIDATOR_WITH_LOCAL_WITH_TLD;
-            } else {
-                return EMAIL_VALIDATOR_WITH_LOCAL;
-            }
-        } else {
-            if (allowTld) {
-                return EMAIL_VALIDATOR_WITH_TLD;
-            } else {
-                return EMAIL_VALIDATOR;
-            }
+           return EMAIL_VALIDATOR_WITH_LOCAL;
         }
-    }
-
-    /**
-     * Returns the Singleton instance of this validator,
-     *  with local validation as required.
-     *
-     * @param allowLocal Should local addresses be considered valid?
-     * @return singleton instance of this validator
-     */
-    public static EmailValidator getInstance(boolean allowLocal) {
-        return getInstance(allowLocal, false);
-    }
-
-    /**
-     * Protected constructor for subclasses to use.
-     *
-     * @param allowLocal Should local addresses be considered valid?
-     */
-    protected EmailValidator(boolean allowLocal, boolean allowTld) {
-        super();
-        this.allowLocal = allowLocal;
-        this.allowTld = allowTld;
+        return EMAIL_VALIDATOR;
     }
 
     /**
@@ -140,7 +95,6 @@ public class EmailValidator implements Serializable {
     protected EmailValidator(boolean allowLocal) {
         super();
         this.allowLocal = allowLocal;
-        this.allowTld = false;
     }
 
     /**
@@ -194,11 +148,8 @@ public class EmailValidator implements Serializable {
         // Domain is symbolic name
         DomainValidator domainValidator =
                 DomainValidator.getInstance(allowLocal);
-        if (allowTld) {
-            return domainValidator.isValid(domain) || domainValidator.isValidTld(domain);
-        } else {
-            return domainValidator.isValid(domain);
-        }
+        return domainValidator.isValid(domain) ||
+                domainValidator.isValidTld(domain);
     }
 
     /**
