diff --git a/src/share/org/apache/commons/validator/EmailValidator.java b/src/share/org/apache/commons/validator/EmailValidator.java
index 1b1b7329..a220cdd1 100644
--- a/src/share/org/apache/commons/validator/EmailValidator.java
+++ b/src/share/org/apache/commons/validator/EmailValidator.java
@@ -38,7 +38,7 @@ import org.apache.oro.text.perl.Perl5Util;
  */
 public class EmailValidator {
 
-    private static final String SPECIAL_CHARS = "\\000-\\037\\(\\)<>@,;:'\\\\\\\"\\.\\[\\]\\177";
+    private static final String SPECIAL_CHARS = "\\(\\)<>@,;:'\\\\\\\"\\.\\[\\]";
     private static final String VALID_CHARS = "[^\\s" + SPECIAL_CHARS + "]";
     private static final String QUOTED_USER = "(\"[^\"]*\")";
     private static final String ATOM = VALID_CHARS + '+';
