diff --git a/src/test/org/apache/commons/validator/EmailTest.java b/src/test/org/apache/commons/validator/EmailTest.java
index a1a3496e..a5780128 100644
--- a/src/test/org/apache/commons/validator/EmailTest.java
+++ b/src/test/org/apache/commons/validator/EmailTest.java
@@ -221,18 +221,6 @@ public class EmailTest extends TestCommon {
 
     }
 
-   /**
-    * Tests the email validation with ascii control characters.
-    * (i.e. Ascii chars 0 - 31 and 127)
-    */
-    public void testEmailWithControlChars() throws ValidatorException {
-        EmailValidator validator = new EmailValidator();
-        for (char c = 0; c < 32; c++) {
-            assertFalse("Test control char " + ((int)c), validator.isValid("foo" + c + "bar@domain.com"));
-        }
-        assertFalse("Test control char 127", validator.isValid("foo" + ((char)127) + "bar@domain.com"));
-    }
-
     /**
      * Write this test according to parts of RFC, as opposed to the type of character
      * that is being tested.
