diff --git a/src/share/org/apache/commons/validator/ValidatorResources.java b/src/share/org/apache/commons/validator/ValidatorResources.java
index 7158337a..2c1e5d5d 100644
--- a/src/share/org/apache/commons/validator/ValidatorResources.java
+++ b/src/share/org/apache/commons/validator/ValidatorResources.java
@@ -54,9 +54,6 @@ import org.xml.sax.Attributes;
  */
 public class ValidatorResources implements Serializable {
 
     /**
      * The set of public identifiers, and corresponding resource names, for
      * the versions of the configuration file DTDs that we know about.  There
@@ -202,14 +199,7 @@ public class ValidatorResources implements Serializable {
      *  Initialize the digester.
      */
     private Digester initDigester() {
-        if (rulesUrl == null) {
-            rulesUrl = ValidatorResources.class.getResource(VALIDATOR_RULES);
-        }
         Digester digester = DigesterLoader.createDigester(rulesUrl);
         digester.setNamespaceAware(true);
         digester.setValidating(true);
