diff --git a/src/share/org/apache/commons/validator/ValidatorResources.java b/src/share/org/apache/commons/validator/ValidatorResources.java
index 7158337a..2c1e5d5d 100644
--- a/src/share/org/apache/commons/validator/ValidatorResources.java
+++ b/src/share/org/apache/commons/validator/ValidatorResources.java
@@ -54,9 +54,6 @@ import org.xml.sax.Attributes;
  */
 public class ValidatorResources implements Serializable {
 
-    /** Name of the digester validator rules file */
-    private static final String VALIDATOR_RULES = "digester-rules.xml";
-
     /**
      * The set of public identifiers, and corresponding resource names, for
      * the versions of the configuration file DTDs that we know about.  There
@@ -202,14 +199,7 @@ public class ValidatorResources implements Serializable {
      *  Initialize the digester.
      */
     private Digester initDigester() {
-        URL rulesUrl = this.getClass().getResource(VALIDATOR_RULES);
-        if (rulesUrl == null) {
-            // Fix for Issue# VALIDATOR-195
-            rulesUrl = ValidatorResources.class.getResource(VALIDATOR_RULES);
-        }
-        if (getLog().isDebugEnabled()) {
-            getLog().debug("Loading rules from '" + rulesUrl + "'");
-        }
+        URL rulesUrl = this.getClass().getResource("digester-rules.xml");
         Digester digester = DigesterLoader.createDigester(rulesUrl);
         digester.setNamespaceAware(true);
         digester.setValidating(true);
