diff --git a/src/main/java/org/apache/commons/validator/GenericTypeValidator.java b/src/main/java/org/apache/commons/validator/GenericTypeValidator.java
index c6bd6c11..4100496f 100644
--- a/src/main/java/org/apache/commons/validator/GenericTypeValidator.java
+++ b/src/main/java/org/apache/commons/validator/GenericTypeValidator.java
@@ -371,38 +371,22 @@ public class GenericTypeValidator implements Serializable {
         }
 
         try {
-            DateFormat formatterDefault = null;
             if (locale != null) {
-                formatterDefault = 
-                   DateFormat.getDateInstance(DateFormat.DEFAULT, locale);
             } else {
                     DateFormat.getDateInstance(
                     DateFormat.SHORT,
                     Locale.getDefault());
-                   DateFormat.getDateInstance(
-                   DateFormat.DEFAULT,
-                   Locale.getDefault());
             }
 
-            formatterDefault.setLenient(false);
 
-            try {
-               date = formatterShort.parse(value);
-            } catch (ParseException e) {
-               date = formatterDefault.parse(value);
-            }
+            date = formatter.parse(value);
         } catch (ParseException e) {
             Log log = LogFactory.getLog(GenericTypeValidator.class);
             if (log.isDebugEnabled()) {
                 log.debug("Date parse failed value=[" + value  + "], " +
