diff --git a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
index 22ef37dd..19705983 100644
--- a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
+++ b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
@@ -141,11 +141,6 @@ public class EmailValidatorTest extends TestCase {
         assertTrue(validator.isValid("me@att.net")); // Make sure TLD is not the cause of the failure
     }
 
-    public void testVALIDATOR_278() {
-        assertFalse(validator.isValid("someone@-test.com"));// hostname starts with dash/hyphen
-        assertFalse(validator.isValid("someone@test-.com"));// hostname ends with dash/hyphen
-    }
-
     /**
     * Tests the email validation with commas.
     */
